docs/Learning-Environment-Design-Agents.md

```diff
@@ -112,7 +112,13 @@ Rotations and angles should also be normalized. For angles between 0 and 360 deg
 Camera observations use rendered textures from one or more cameras in a scene. The brain vectorizes the textures and feeds them into a neural network. You can use camera observations and either continuous feature vector or discrete state observations at the same time.
  
 Agents using camera images can capture state of arbitrary complexity and are useful when the state is difficult to describe numerically. However, they are also typically less efficient and slower to train, and sometimes don't succeed at all.  
-  
+
+To add a visual observation to an agent, click on the `Add Camera` button on the Agent's inspector. Then drag the camera you want to use in the placeholder box. 
```

Comment:
To add a visual observation to an agent, click on the **Add Camera** button on the Agent Inspector. Then drag the camera you want to use to the **Camera** field.